src/js/stores/document.js

```diff
@@ -1011,8 +1014,51 @@ define(function (require, exports, module) {
                 nextDocument = document.set("layers", nextLayers);
 
             this.setDocument(nextDocument, true);
+        },
+
+        /**
+         * Updates a guide with new information
+         * 
+         * @private
+         * @param {{documentID: number, index: number, orientation: string, position: number}} payload
+         */
+        _handleGuideMoved: function (payload) {
+            var documentID = payload.documentID,
+                index = payload.index,
+                document = this._openDocuments[documentID],
+                orientation = payload.orientation,
+                position = payload.position;
+
+            var nextGuide = document.guides.get(index)
+                .merge({
```

Comment:
I think .merge should be indented further IF you are going to include other vars below in the same stmt.

also this might throw if there is no guide at `index`